webui/react/src/quarantineTests.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+type ItStandin = (name: string, fn: (...args: any) => any, timeout?: number | undefined) => void;
+
+export const quarantinedIt = (): ItStandin => {
+  return process.env.QUARANTINED ? it.skip : it;
+};
```

Comment:
**suggestion:** I saw the usage and realize it's not quite streamlined. I think with the goal of using `quarantinedIt` to identical to `it` and `it.skip` something like this would be better:
```
export const quarantinedIt = (
  name: string,
  fn?: jest.ProvidesCallback,
  timeout?: number,
): void => {
  return process.env.QUARANTINED
    ? it.skip(name, fn, timeout)
    : it(name, fn, timeout);
};
```

Reply:
oh I'm also noticing the it and it.skip gotta get reversed (?)